Dockerfile

```diff
@@ -41,7 +41,7 @@ WORKDIR /app/
 COPY cpanfile cpanfile.snapshot ./
 RUN \
     --mount=type=cache,target=/root/.perl-cpm,sharing=private \
-<<EOT /bin/bash -euo pipefail
+<<EOT
```

Comment:
Wondering why the bash strict mode is getting removed?

Reply:
It's already set with a `SHELL` command. I've gone back and forth on how I wanted to handle this, but for now the `SHELL` command seems fine.